papaparse.js

```diff
@@ -1092,7 +1099,7 @@ License: MIT
 		{
 			_paused = true;
 			_parser.abort();
-			_input = _input.substr(_parser.getCharIndex());
+			_input = _input.substring(_parser.getCharIndex());
```

Comment:
Why changing this?

Reply:
Refactored to not use substr which is a legacy function and should be treated as depreciated.

Reply:
Ok, could you please move this to a separate PR which does not modify the test suite?